_entries/2016-05-04-hyperlinks.md

```diff
@@ -35,16 +35,20 @@ Not this
 
 Don’t use meaningless terms such as ‘click here’, ‘read more’ or ‘useful links’.
 
-### Drop the http://
+### Drop the https://
 
-When referencing a website, drop the `http://` prefix.
+You don't need to write `http://` or `https://` when referencing a website.
+
+If the site has a URL route without `www` you don't need to write it when you reference it. But keep the `www` if there is only a redirect in place.
```

Comment:
Could we rewrite line 42 towards this? "If the site can be accessed without `www` in the address, you don't…"

Main reason being 'URL route' should probably be 'hostname' to be technically correct.

I wonder if the guidance here could be structured around what the canonical URL for a piece of content is i.e. "Ensure links are as direct as possible. Avoid linking to pages that redirect to another, final page, where possible."

Is that too jargon-y?